tests/foreman/cli/test_product.py

```diff
@@ -190,3 +192,62 @@ def test_negative_create_with_label(self):
                         u'name': product_name,
                         u'organization-id': self.org['id'],
                     })
+
+    @tier2
+    def test_product_list_with_default_settings(self):
+        """Listing product of an organization apart from default organization using hammer
+         does not return output if a defaults settings are applied on org.
+
+        :id: d5c5edac-b19c-4277-92fe-28d9b9fa43ef
+
+        :BZ: 1745575
+
+        :expectedresults: product/reporsitory list should work as expected.
+
+        """
+        default_product_name = gen_string('alpha')
+        non_default_product_name = gen_string('alpha')
+        default_org = make_org()
+        non_default_org = make_org()
```

Comment:
I wonder if we really need to create any organizations in this test. One org is created in the setup and there's also Default org so that is two. I'd say reusing those would speed up the test execution

Reply:
The reason I created separate or is that there are multiple other products are already created in that org. I wanted to avoid that, even if we add the default or to module org won't make the problem. let me test that gets back to you